src/cisco_gnmi/nx.py

```diff
@@ -154,19 +288,18 @@ def subscribe_xpaths(
 
     def parse_xpath_to_gnmi_path(self, xpath, origin=None):
         """Attempts to determine whether origin should be YANG (device) or DME.
-        Errors on OpenConfig until support is present.
         """
-        if xpath.startswith("openconfig"):
-            raise NotImplementedError(
-                "OpenConfig data models not yet supported on NX-OS!"
-            )
         if origin is None:
             if any(
-                map(xpath.startswith, ["Cisco-NX-OS-device", "/Cisco-NX-OS-device"])
+                map(xpath.startswith, [
+                    "Cisco-NX-OS-device",
+                    "/Cisco-NX-OS-device",
+                    "cisco-nx-os-device",
+                    "/Cisco-nx-os-device"])
             ):
                 origin = "device"
                 # Remove the module
                 xpath = xpath.split(":", 1)[1]
             else:
-                origin = "DME"
```

Comment:
Okay with this given the DME XPath handling is somewhat broken in this version of the lib per #43